docs/aws/README.md

```diff
@@ -107,6 +107,8 @@ spec:
     name: ${CLUSTER_NAME}
     caBundle: $(aws eks describe-cluster --name ${CLUSTER_NAME} --query "cluster.certificateAuthority.data" --output json)
     endpoint: $(aws eks describe-cluster --name ${CLUSTER_NAME} --query "cluster.endpoint" --output json)
+  utilization:
+    ttlSeconds: 0
```

Comment:
I think we want to suggest the ttlSecondsUtilization here to be 300, right? Setting it to 0 would cause a lot of thrashing.

This should also be a top level parameter now, right?

Reply:
From @rothgar's PR, making this 30.